tachyon/zk/lookup/log_derivative_halo2/prover_impl.h

```diff
@@ -232,7 +232,7 @@ BlindedPolynomial<Poly, Evals> Prover<Poly, Evals>::CreateGrandSumPoly(
   // 1 / τ(X)
   ComputeLogDerivatives(compressed_table, beta, storage.table_log_derivatives);
 
-  std::vector<F> grand_sum(n);
+  std::vector<F> grand_sum(n, F::Zero());
```

Comment:
It seems like loop below fills the values up to usable rows + 1. just initialize the values for remaining indicies. But just add a comment that why this is safe.